### ISSUE TITLE
go get fails on master

### ISSUE THREAD

**Issue by raesene** [controlplaneio/kubesec#58]
Looking at installing from master fails at the moment.  Output of  `go get -u github.com/controlplaneio/kubesec/cmd/kubesec`

```
# github.com/controlplaneio/kubesec/pkg/ruler
hostdocs/Development/go/src/github.com/controlplaneio/kubesec/pkg/ruler/ruleset.go:277:41: cannot use "resource.json" (type string) as type *kubeval.Config in argument to kubeval.Validate
```

Looking at `https://github.com/instrumenta/kubeval/commits/master/kubeval/config.go` looks like there's been some changes in how they handle config...

**Comment by sublimino** [comment]
Thanks for the report @raesene : ]

I'm moving house this weekend so will have a look next week.

**Comment by mike-stewart** [comment]
Any plans to fix this? I get the same error when trying to use `go get`.

I get a similar error when trying to build from source:
```
go build -a -o ./dist/kubesec ./cmd/kubesec/*.go
# github.com/controlplaneio/kubesec/pkg/ruler
../../go/src/github.com/controlplaneio/kubesec/pkg/ruler/ruleset.go:286:41: cannot use "resource.json" (type string) as type *kubeval.Config in argument to kubeval.Validate
make: *** [build] Error 2
```

When combined with #53, it seems like there is no way to build or download the kubesec binary. Any workaround?

**Comment by sublimino** [comment]
@mike-stewart we'll merge a fix for #53 in #68 shortly, then migrate from dep to gomodules to fix this. Sorry for the latency! :pray:

**Comment by 06kellyjac** [comment]
The new release has moved to go modues

I've tried `go get` in a go 1.13 container and it worked. Please comment if it doesn't work and I'll re-open the issue.

```
docker run -it golang:1.13-alpine
/go # apk add -U git
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
(1/5) Installing nghttp2-libs (1.39.2-r0)
(2/5) Installing libcurl (7.66.0-r0)
(3/5) Installing expat (2.2.8-r0)
(4/5) Installing pcre2 (10.33-r0)
(5/5) Installing git (2.22.0-r0)
Executing busybox-1.30.1-r2.trigger
OK: 21 MiB in 20 packages
/go # go get -u github.com/controlplaneio/kubesec/cmd/kubesec
/go # ls bin/
kubesec
/go # ./bin/kubesec version
version unknown
git commit unknown
build date unknown
```
